pkg/resourcemanager/mysql/mysqluser/mysqluser.go

```diff
@@ -7,27 +7,23 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"strings"
 
-	mysql "github.com/Azure/azure-sdk-for-go/services/mysql/mgmt/2017-12-01/mysql"
+	mysqlmgmt "github.com/Azure/azure-sdk-for-go/services/mysql/mgmt/2017-12-01/mysql"
+
 	"github.com/Azure/azure-service-operator/pkg/helpers"
 	"github.com/Azure/azure-service-operator/pkg/resourcemanager/config"
+	"github.com/Azure/azure-service-operator/pkg/resourcemanager/mysql"
 	mysqldatabase "github.com/Azure/azure-service-operator/pkg/resourcemanager/mysql/database"
 	"github.com/Azure/azure-service-operator/pkg/secrets"
 
-	"github.com/Azure/azure-service-operator/api/v1alpha1"
 	"k8s.io/apimachinery/pkg/runtime"
 
+	"github.com/Azure/azure-service-operator/api/v1alpha1"
+
 	_ "github.com/go-sql-driver/mysql" //mysql drive link
 	"k8s.io/apimachinery/pkg/types"
```

Comment:
Same here - these should go up into the external packages group.

Reply:
Fixed. For some reason goland does an ok job of fixing these up automatically but it's not perfect (sometimes it doesn't merge some blocks when it should).